space_packet_parser/xtce/parameter_types.py

```diff
@@ -238,7 +241,8 @@ def from_xml(
             ns: dict,
             tree: Optional[ElementTree.Element] = None,
             parameter_lookup: Optional[dict[str, any]] = None,
-            parameter_type_lookup: Optional[dict[str, any]] = None
+            parameter_type_lookup: Optional[dict[str, any]] = None,
+            container_lookup: Optional[dict[str, any]] = None
```

Comment:
These seem repetitive if they are rarely used. Would it make sense to pass in a `definition` here instead so you can consolidate all 3 of those items onto the parent object that holds all of those. That may make testing harder though?

Reply:
Yeah, these are all on the ABC to satisfy Liskov substitution. The argument for keeping these kwargs separate is to make it explicit which methods use which of the three lookups. And yes, it does make things easier to test if we can just pass lists rather than a full XtcePacketDefinition object.